pandas/_libs/window/indexers.pyx

```diff
@@ -58,6 +59,9 @@ def calculate_variable_window_bounds(
     if closed in ['left', 'both']:
         left_closed = True
 
+    if index[num_values - 1] < index[0]:
```

Comment:
are we checking if len > 0 before this?

Reply:
i guess this is done in some caller method, as we access `start[0]` and start is initialized to the same length as `index`. (checked by a test too)